### PR TITLE
PD-13507 SidePanel: fix bodyScroll

### PR DESCRIPTION
**Pull request by AndreyChernykh**
### 🛠 Purpose

After closing the SidePanel, <body> still has `overflow: hidden;` 🐞  

---

### ✏️ Notes to Reviewer

1. `useBodyOverflow` -> `< LockBodyScroll />`
2.  `useEscapeKey` -> `< EscListener  />`

---

### 🖥 Screenshots

---

### CODE REVIEW

File: packages/SidePanel/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paprika/sidepanel",
-  "version": "0.1.14",
+  "version": "0.2.0",
```
Comment (nahumzs) [inline packages/SidePanel/package.json]: you shouldn't bump this one, we will do it when we release via lerna.

File: packages/SidePanel/src/components/EscListener.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const propTypes = {
+  /** Callback triggered on ESC key down */
+  onEscKeydown: PropTypes.func.isRequired,
+};
+
+function EscListener({ onEscKeydown }) {
```
Comment (alexzherdev) [inline packages/SidePanel/src/components/EscListener.js]: This looks exactly like a custom hook, not a component

File: packages/SidePanel/src/components/EscListener.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
```
Comment (nahumzs) [inline packages/SidePanel/src/components/EscListener.js]: `fyi` I think this change might conflict with @DanilAgafonov takeover migration PR, but might not be that bad.

File: packages/SidePanel/src/components/LockBodyScroll.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+function LockBodyScroll() {
```
Comment (alexzherdev) [inline packages/SidePanel/src/components/LockBodyScroll.js]: Not a fan of repeating this file in every component. Perhaps something we could add into `helpers`?

Comment (nahumzs) [reply packages/SidePanel/src/components/LockBodyScroll.js]: let's merge this one, and maybe @DanilAgafonov can consolidate these two into the helpers in his PR.

**Pull request merged**